client/src/index.js

```diff
@@ -46,6 +52,7 @@ const Routes = () => {
           <Route path="/" exact component={Home} />
           <Route path="/about/" component={About} />
           <Route path="/status/" component={Status} />
+          <Route path="/UserRegisterForm/" component={UserRegisterForm} />
```

Comment:
Can you change the path to match the same case and format used for other interpreter sign up route, i.e. `/sign-up-user`

Reply:
ok i will change it